src/styles/components/_Logo.scss

```diff
@@ -11,32 +11,32 @@
 	width: 292px;
 	height: 56px;
 
-	&.Logo-red { background-image: url( "../images/logo-red.svg" ); }
-	&.Logo-white { background-image: url( "../images/logo-white.svg" ); }
+	&.HMLogo-red { background-image: url( "../images/logo-red.svg" ); }
+	&.HMLogo-white { background-image: url( "../images/logo-white.svg" ); }
 
-	&.Logo-alt {
+	&.HMLogo-alt {
```

Comment:
The alt can be unnested/unoverloaded, the colour will need to be overloaded for the logo type variants. (Or we could go with `-alt-red`, etc but ugh.